src/java.base/share/classes/java/lang/reflect/ClassFileFormatVersion.java

```diff
@@ -49,6 +49,47 @@
  */
 @SuppressWarnings("doclint:reference") // cross-module links
 public enum ClassFileFormatVersion {
+    /*
+     * Summary of class file format evolution; previews are listed for
+     * convenience, but they are not modeled by this enum.
+     *  1: InnerClasses, Synthetic, Deprecated attributes
+     *  2: ACC_STRICT modifier
+     *  3: no changes
+     *  4: no changes
```

Comment:
The version mapping can be tricky here: is "1" 1.0 plus 1.1? I presume "2" is 1.2, "3" is 1.3.x, "4" is 1.4.x

Might be useful to include that actual JVMS classfile version numbers for ease of reference back to JVMS.

Reply:
How about versions like 1.1, ... 1.8, 9, ... which follows the since versions of libraries and should have no ambiguity?  I will commit if everyone agrees.